src/applications/static-pages/sharable-links/sharableLink.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from 'react';
```

Comment:
This feels a big heavy handed to me for a tooltip. Looking at the usage could we use relative positioning instead?

Reply:
if you can point me to where else in the code base we have a tool tip that is dynamically aligned, let me know. 
fyi this **_is_** using relative positioning. except for when the tool tip is right aligned (then it is absolute). if i were to default to relative positioning, i would still need to set left and bottom px values dynamically.

Reply:
Probably something like this: https://codepen.io/timwright12/pen/abpjZZz

Reply:
yeah, so the gotcha here is that the positioning of the tooltip (left or right) is based on dynamic content length (the parent h3 which comes from the CMS). in this PR I am calculating the width of that element, and then positioning accordingly

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/16759/files#diff-c3201682c3d0a2103826f66b5bcfd1effa362c465b0c7b9a63859a53feb7824aR112

Reply:
If you're just talking about keeping it flush the to right or left the CSS version does accomplish that https://codepen.io/timwright12/pen/abpjZZz still no good?